visualizer/flame-graph.js

```diff
@@ -138,7 +138,7 @@ class FlameGraph extends HtmlContent {
       element: this.d3Chart.node(),
       cellHeight: this.cellHeight,
       collapseHiddenNodeWidths: true,
-      minHeight: window.screen.availHeight,
+      minHeight: flameWrapper.d3Element.node().clientHeight,
```

Comment:
Only sets it on page load. Maybe we need something in d3-fg like `chart.minHeight` equivalent to `chart.height` for updating the minimum height on `resize`.

Reply:
OK cool, cheers @AlanSl. Will try and replicate and investigate 👍